### ISSUE TITLE
How can I finetune the VAE of stable diffusion?

### ISSUE THREAD

**Issue by eeyrw** [CompVis/stable-diffusion#425]
Any hints or suggestion?

**Comment by BenjaminIrwin** [comment]
@eeyrw did you have any luck finding out more about this?

**Comment by eeyrw** [comment]
> @eeyrw did you have any luck finding out more about this?

No. I closed the issue just because of duplication question. But after that I actually have some tries, but result to failure. You may check this [repo](https://github.com/CompVis/latent-diffusion). There are some instructions about how to train auto encoder. I tried but stuck at this [issue](https://github.com/CompVis/latent-diffusion/issues/176).

**Comment by keyu-tian** [comment]
@BenjaminIrwin @eeyrw i think im doing right on VQVAE finetuning. Just using https://github.com/CompVis/taming-transformers . I use a very small learning rate (2e-6 for a batch size of 512) and it works for me.

**Comment by MaybeShewill-CV** [comment]
@keyu-tian Hi may i wonder have you successfully interegated your ft-vae model with latent diffusion?

**Comment by Leminhbinh0209** [comment]
Try my implementation here: https://github.com/Leminhbinh0209/FinetuneVAE-SD
